src/UserFunctions.java

```diff
@@ -30,16 +34,17 @@ public UserFunctions() {
      * @return If the quit command was given, returns false; otherwise, returns true.
      */
     public bool parseCommand(String input) {
```

Comment:
In Java, boolean needs to be used.

```suggestion
    public boolean parseCommand(String input) {
```